concent_api/core/payments/backends/sci_backend.py

```diff
@@ -20,6 +23,24 @@ class TransactionType(Enum):
     FORCE = 'force'
 
 
+def handle_sci_synchronization(sci_function: Callable) -> Callable:
+
+    @wraps(sci_function)
+    def wrapper(*args: Any, **kwargs: Any) -> None:
+        if PaymentInterface().is_synchronized():  # pylint: disable=no-member
+            try:
+                return sci_function(*args, **kwargs)
+            except ValueError as exception:
+                if "There are currently no blocks after" in str(exception):
+                    return []
+                else:
+                    raise
+        else:
+            raise SCINotSynchronized()
```

Comment:
You may add `'SCI is currently not synchronized'` message to the exception body.

Reply:
Done